docs/home/whats-new-2-13.md

```diff
@@ -73,9 +73,14 @@ Added `suspend`/`stop` to [Submitting Workloads via HTTP/REST](../developer/clus
 
 <!-- RUN-9521/9522  Provide a description in CLI when command fails -->
 
+
+<!-- RUN-10287/10317/10313-10851 Show Node pools priority list according to workspace policy -->
+When there is a workspace policy in place, the node pool priority list in *Workspaces* is the same as the policy, and is editable when the workspace is built from scratch. The list will not be editable if the policy dictates that the list is locked. If a policy has not been enabled, then the priority list will come from either a template, or the project. 
```

Comment:
I think it is written in a confusing way, the message needs to be:
The selection of node pools is now incorporated in the workspace submission page, this allows data scientists to quickly determine the list of node pools to try by dragging & dropping to their desired order of priority.

Moreover, when the node pools priority list is locked by admin's policy this list will be freezed & the data scientist will be able to edit it even if the workspace is created from the template & duplicated

Reply:
See the new changes in the "Files Changed" link at the top.